### DISCUSSION TITLE
How do I restore my lightning channels funds?

### DISCUSSION THREAD

**Discussion by Ganzosupremo** [Ride-The-Lightning/RTL#828] [Q&A] [answered]
Hello
I've a problem with my node and I have to reinstall everything, I backup everything before that, I was able to recover my on-chain funds, but I'm not able to how to recover my Lightning funds. RTL says that I need to put the channel.backup file in this direction: /mnt/hdd/mynode/rtl_backup//restore. But I don't even know how to access that direction, so if someone could please tell me how do I access that folder, I would appreciate it so much.
I use Mynode as the OS for my node, btw.

**Comment by saubyk** [comment, marked as answer]
@Ganzosupremo 
You would need to open up the terminal on your node and access the file system.

Did you take the channel backup from RTL? Or is it the LND channel back up file?

**Reply by Ganzosupremo** [reply]
I took the backup from RTL

**Reply by Ganzosupremo** [reply]
And how do I open the terminal in my node? with powershell or some other? 
sorry I'm relative new to this.

**Reply by Ganzosupremo** [reply]
I've access the system with Powershell, now, how do I add the backup to the /mnt/hdd/mynode/rtl_backup//restore, which command adds it?

**Reply by saubyk** [reply]
Hi @Ganzosupremo, sorry I missed this. Try these steps:

1. Create a restore folder in your folder backup location, as specified in the RTL config file.
2. Place the channel backup file in the restore folder.
3. Access the Restore tab under the Backup page of RTL.
4. RTL will list the options to restore funds from the all channel file or individual channel backup file.
5. Click on the Restore icon on the grid to restore the funds.
6. Once the restore function is executed successfully, RTL will rename the backup file and it will be removed from the UI.
7. Restore function will force close the channels and recover the funds from them.
8. The pending close channels can be viewed on the Pending page under Channels.
9. The corresponding pending on-chain transactions can also be viewed on the List Transactions page under LND Wallet.
10. Once the transactions are confirmed, the channels funds will be restored to your LND Wallet.

**Reply by Ganzosupremo** [reply]
Thanks @saubyk I think I understood the most parts except the first one. So, how do I exactly create a restore folder in the RTL config file?
because there isn't even a visible option to access the RTL config file. Can you please tell how I access the config file, create a folder and move the channel backup there.
Sorry for all the questions and the trouble, I'm not a programmer and this is the first time I do this kind of thing.

**Comment by awhitedog69** [comment]
Did you ever resolve this issue? I am having exact same problem

**Reply by ShahanaFarooqui** [reply]
@awhitedog69 & @Ganzosupremo, 

Rather than RTL-Config.json, you can verify your `Backup Folder Location` from `backup`/`restore` tabs:
![Screenshot from 2022-11-07 08-47-53](https://user-images.githubusercontent.com/23240721/200367568-213dcdd3-4992-467d-8d6a-59dc3d612415.png)

Once you know your backup location, please follow @saubyk steps from:
https://github.com/Ride-The-Lightning/RTL/discussions/828#discussioncomment-1433075
